docs/contributing/triaging-github-issues.md

```diff
@@ -3,7 +3,103 @@ title: Triaging GitHub Issues
+- Unrelated issues are resolved swiftly so we don't spend too much time on them
+
+## What is unique to Gatsby?
+
+Gatsby is unique among most open source projects because:
+
+- Gatsby integrates with many third party tools (WordPress, Drupal, Contentful etc) via source plugins and hence the typical scope of issues is widened significantly
+- Gatsby aims to be really beginner friendly (we want to be the new way someone gets started with web development) and this means that we need to accommodate a wide range of skill levels
+- At Gatsby, we've defined a couple of metrics that we measure to ensure we're responsive and helpful to our open source community
+
+## How do we do first touch maintenance?
+
+### General Guidelines
+
+- **Be empathic.** The author of an issue might be asking something that’s obvious to you but that doesn’t mean it’s obvious to them - it's important to consider the issue from the author’s viewpoint. People often remember how you make them feel, not what you told them.
+- **Add context.** When answering an issue, it can be useful to link to existing documentation, issues, PRs, or provide related context. This means the issue can serve as a reference to future readers.
+- **Encourage community contributions.** Getting people involved makes a huge impact. It’s often worth spending the time to write up a task as a `good first issue` instead of fixing the issue yourself. This can provide a low friction way for someone to get more involved in open source!
+- **Give issue authors time to close their own issues.** Sometimes, it might feel like an issue is resolved but the author could have follow up questions. It's usually best to give them a day or two to close the issue themselves.
+
+### Labelling
+
+Labelling helps group issues into manageable sets and also improves searchability and scannability. We have a set of labels that we use to group issues based on their type and status. While we want to limit adding too many labels, feel free to add one if it seems relevant and helps with this grouping!
+
+It's nice to update labels as the state of an issue changes or if for example, a question becomes a feature request or a bug report
+
+### Resolution Flow Chart
+
+https://whimsical.co/QvuMgo31T2C3xcWbou8xhy
+
+### Saved Replies
+
+https://github.com/orgs/gatsbyjs/teams/admin/discussions/3
+
+## Bot
+
+We have a bot that helps us automate some aspects:
+
+- Issues with a question mark in their title or starting with "how" are automatically labelled as questions
+- Issues with an empty body are closed
+- Issues with no activity are marked stale after 20 days. They are then closed after another 10 days unless anyone comments saying not stale or adds the not stale label
+
+## Frequently Asked Questions
+
+> When do I do a demo for an issue?
+
+When a feature or pattern is not documented, it may be nice to make a demo to add clarity for the author and help future readers as well.
```

Comment:
I'm a little confused by this?

By demo, do you mean reproduction, or do you mean something else here?

If you mean reproduction we should tweak this to be a little stronger, ideally--we'd always have a reproduction if it's able to be reproduced and is a bug.

Also https://gatsby.dev/reproduction is worth cross-linking here.

Reply:
I think this is separate.  I'll add a section on bug reproductions.